.github/workflows/tests.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
       run: |
         pip install wheel
         pip install -e ".[tests]"
+    - name: check env
+      run: env
```

Comment:
since it's a public project does it mean that we exposed all variables by doing this check (in CI logs)?

Reply:
Yes, probably not wisest on my part, all logs have been deleted